.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,5 +1,17 @@
 blank_issues_enabled: false
 contact_links:
-  - name: MOSS Discord Server
+  - name: Create help request
+    url: https://github.com/EssentialsX/Essentials/discussions/new?category=q-a
+    about: Create a support ticket to get help from developers.
+  - name: Suggest a feature
+    url: https://github.com/EssentialsX/Essentials/discussions/new?category=ideas-and-feature-suggestions
+    about: Suggest new features for EssentialsX!
+  - name: View previous help requests
+    url: https://github.com/EssentialsX/Essentials/discussions/categories/q-a
+    about: Visit the help forum to search for existing help requests
+  - name: Discuss existing feature suggestions
+    url: https://github.com/EssentialsX/Essentials/discussions/categories/ideas-and-feature-suggestions
+    about: Visit the help forum to search for existing help requests
```

```suggestion
    about: Visit the suggestions forum to discuss existing feature suggestions.
```
i'm bad at words - the actual point is the accidental unchanged copy-paste